app/utils/consts.js

```diff
@@ -3,17 +3,31 @@ const RUNNER_MANDATORY_VARIABLES = [
     'TEST_ID',
     'PREDATOR_URL',
     'DURATION',
-    'ARRIVAL_RATE',
     'JOB_ID',
     'RUN_ID'
 ];
 
+const LOAD_TEST_MANDATORY_VARIABLES = [
+    'ARRIVAL_RATE'
+];
+
+const FUNCTIONAL_TEST_MANDATORY_VARIABLES = [
+    'ARRIVAL_COUNT'
+];
+
 const PLUGINS_MANDATORY_VARIABLES = {
     influx: ['host', 'database', 'username', 'password'],
     prometheus: ['push_gateway_url']
 };
 
+const LOAD_TEST = 'load_test';
+const FUNCTIONAL_TEST = 'functional_test';
+
 module.exports = {
+    FUNCTIONAL_TEST,
+    LOAD_TEST,
+    LOAD_TEST_MANDATORY_VARIABLES,
+    FUNCTIONAL_TEST_MANDATORY_VARIABLES,
     RUNNER_MANDATORY_VARIABLES,
     PLUGINS_MANDATORY_VARIABLES
 };
```

Comment:
We have the same logic for enforcing mandatory variables in both `predator` and `predator-runner`, a package might come in handy in the future.

Reply:
true, but we plan to refactor runner to pull these variables from the `job` and not as environment variables, then we won't need the validations